controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/ApplicationSerializer.java

```diff
@@ -226,8 +228,16 @@ private void zoneIdToSlime(ZoneId zone, Cursor object) {
         object.setString(regionField, zone.region().value());
     }
 
-    private void versionsToSlime(List<ApplicationVersion> versions, Cursor object) {
-        versions.forEach(version -> toSlime(version, object.addObject()));
+    private void versionsToSlime(Application application, Cursor object) {
+        // Add all deployed versions as well
```

Comment:
And this'll be removed later, once we switch to using only the build number+direcly deployed bit as deployment keys :)